src/components/TableOfContents/index.tsx

```diff
@@ -14,7 +14,7 @@ import { hasKey } from '#utils/common';
 
 import styles from './styles.scss';
 
-const noOp = () => {};
+const noOp = () => ({});
```

Comment:
This doesn't look correct. noop should return nothing.
This change will make noop return an empty object;